frontend/apps/app/components/SessionDetailPage/components/Chat/components/TimelineItem/components/VersionMessage/VersionMessage.tsx

```diff
@@ -32,7 +24,7 @@ type StatusClass =
   | 'statusUnknown'
 
 const parsePatchOperations = (
-  patch: Json,
+  patch: Database['public']['Tables']['building_schema_versions']['Row']['patch'],
```

Comment:
[nits]
Using the `Tables` type might make the code a bit simpler.

```suggestion
  patch: Tables<'building_schema_versions'>['patch'],
```